src/js/apps/patients/sidebar/patient-sidebar_app.js

```diff
@@ -0,0 +1,28 @@
+import { map } from 'underscore';
+import Radio from 'backbone.radio';
+
+import App from 'js/base/app';
+
+import { LayoutView } from 'js/views/patients/sidebar/patient/patient-sidebar_views';
+
+export default App.extend({
+  beforeStart({ patient }) {
+    const patientModel = Radio.request('entities', 'fetch:patients:model', patient.id);
```

Comment:
I don't think you really need to fetch the patient model as it'll be in the click event of the row.  We can just pass the model to this app and use it.